src/devices/tuya.ts

```diff
@@ -4268,6 +4268,29 @@ export const definitions: DefinitionWithExtend[] = [
         },
         whiteLabel: [tuya.whitelabel("AVATTO", "ZWSM16-3-Zigbee", "3 gang switch module", ["_TZ3000_hbic3ka3"])],
     },
+    {
+        fingerprint: tuya.fingerprint("TS0003", ["_TZ3000_ju82pu2b"]),
+        model: "TS0003_switch_3_gang_simple",
```

Comment:
```suggestion
        model: "TW-03",
```

and remove the `whitelabel`